src/main/kotlin/com/hypto/iam/server/db/repositories/PasscodeRepo.kt

```diff
@@ -0,0 +1,53 @@
+package com.hypto.iam.server.db.repositories
+
+import com.hypto.iam.server.db.Tables.PASSCODES
+import com.hypto.iam.server.db.tables.pojos.Passcodes
+import com.hypto.iam.server.db.tables.records.PasscodesRecord
+import java.time.LocalDateTime
+import java.util.UUID
+import org.jooq.impl.DAOImpl
+
+object PasscodeRepo : BaseRepo<PasscodesRecord, Passcodes, UUID>() {
```

Comment:
Rather than mandating passcode as UUID type, make the type as string so that SMS OTP codes can also be stored in the same passcode table.